### ISSUE TITLE
markupToText does not strip content of <script> tags

### ISSUE THREAD

**Issue by BernhardBaumrock** [processwire/processwire-issues#1546]
### Short description of the issue

I'm using markupToText() to display search results to my users and was quite surprised to see some javascript snippets there...

### Expected behavior

markupToText() should strip tags + content of `<script>some code here</script>` completely, at least there should be an option to do so

### Actual behavior

```php
$text = "<h1>foo</h1><script>alert('foo')</script>";
db($sanitizer->getTextTools()->markupToText($text));
```
![image](https://user-images.githubusercontent.com/8488586/159534759-97fbafa0-4f84-409e-a7fc-316c6fa75553.png)

**Comment by BernhardBaumrock** [comment]
This is my workaround:
```php
$text = preg_replace('#<script(.*?)>(.*?)</script>#is', '', $text);
```

**Comment by ryancramerdesign** [comment]
Thanks @BernhardBaumrock I think this makes sense, as the inner content of script tags isn't likely to be useful in any markupToText scenarios I can think of. I've added this, as well as added style and object tags to the mix as well, and the ability to turn it off (just in case someone wants it), or specify your own tags (see the `clearTags` option). I went with an approach to ensure that it can handle nested script (or other) tags, even if that seems like a rare case.
